site/diagrams/sequenceDiagrams/command_control_concept_http.txt

```diff
@@ -0,0 +1,18 @@
+participant Device
+participant HTTP Adapter
+participant AMQP 1.0 Network
+participant Application
+
+Device->HTTP Adapter: 1. send event (hono-ttd=30)
+activate HTTP Adapter
+HTTP Adapter->Application: 2. connected notification (ttd=30)
+activate Application
+Application->HTTP Adapter: 3. transfer("control/TENANT/4711","light","ON")
+HTTP Adapter->Device: 4. event response("light","ON",${reqId})
+deactivate HTTP Adapter
+note over Device
+Command response in next operation
+end note over
+Device->HTTP Adapter: 5. /control/res("hono-cmd=light","ON",${reqId})
```

Comment:
IMHO this request will need to include the request ID (in the header or as a path parameter) and will include the response in the payload, right?
Something like:
`HTTP request("POST", "/control/res/${reqId}", "lumen=100")`

Reply:
Do you prefer path over parameter?

Reply:
I believe that devices will have an easier time putting the information into the path than setting headers. This would also be more consistent with the way the info is conveyed in the topic name when using MQTT, wouldn't it?

Reply:
I agree with the latter and also the reqId feels like a recourse part.